partials/tabs/projects.html

```diff
@@ -26,6 +26,60 @@
       </div>
     </div>
   </section>
+
+  <div id="modal1" class="modal">
+    <div  class="modal-content-bear">
+        <div class="col-md-4 profile">
+            <div class="project-name">
+                {{currentProject.name}}
+            </div>
+            <div class="project-detail-element  reqs-proj">
+                <div ng-show="currentProject.requirements.length>0" class="small-heading">REQUIREMENTS</div>
+                <li class="pr-element-detail milestones" ng-repeat="req in currentProject.requirements">{{ req }}</li><br>
+            </div>
+
+            <div class="project-detail-element">
+                <div ng-show="currentProject.initiatives.length>0" class="small-heading">INITIATIVES</div>
+                <span class="pr-element-detail chip" ng-repeat="initiative in currentProject.initiatives">{{ initiative }}</span><br>
+            </div>
+
+            <div class="project-detail-element">
+                <div ng-show="currentProject.tags.length>0" class="small-heading">TAGS</div>
+                <span class="pr-element-detail chip" ng-repeat="tags in currentProject.tags">{{ tags }}</span>
+            </div>
+
+             <div class="project-detail-element">
+                <div ng-show="currentProject.difficulty" class="small-heading">DIFFICULTY</div>
+                <span class="pr-element-detail chip">{{ currentProject.difficulty }}</span>
+            </div>
+
+              <div class="project-detail-element">
+                <div ng-show="currentProject.mentors.length>0" class="small-heading">MENTORS</div>
+                <span class="pr-element-detail chip" ng-repeat="mentor in currentProject.mentors">@{{ mentor }}</span>
+            </div>
+            <br>
+        </div>
+
+        <div class="col-md-8 dashboard">
+            <div class="project-description">
+                {{currentProject.desc}}
+            </div>
+            <div class="row milestones-row">
+            <div class="milestones medium-heading">MILESTONES</div>
+            <div class="small-heading">GSOC 2017 CODING STARTS</div>
+             <li class="milestone" ng-repeat="milestone in currentProject['milestones']['GSoC 2017 Coding Starts']">{{ milestone }}</li>
```

Comment:
uhm I'd like this to be dynamic for all possible milestones so we're not bound to GSoC with this system